File: Polymorphism.cs
using System;

class Shape {
    public virtual void Draw() {
        Console.WriteLine("Drawing Shape");
    }
}

class Circle : Shape {
    public override void Draw() {
        Console.WriteLine("Drawing Circle");
    }
}

class Square : Shape {
    public override void Draw() {
        Console.WriteLine("Drawing Square");
    }
}

class Program {
    static void Main() {
        Shape[] shapes = { new Circle(), new Square() };
        foreach (var shape in shapes) {
            shape.Draw();
        }
    }
}

File: Polymorphism.php
<?php
class Shape {
    public function draw() {
        echo "Drawing Shape\n";
    }
}

class Circle extends Shape {
    public function draw() {
        echo "Drawing Circle\n";
    }
}

class Square extends Shape {
    public function draw() {
        echo "Drawing Square\n";
    }
}

$shapes = [new Circle(), new Square()];
foreach ($shapes as $shape) {
    $shape->draw();
}
?>
